test/allowlist_for_publicAPI.json

```diff
@@ -1967,6 +1967,51 @@
     "Transformer",
     "Tuple"
   ],
+  "torch.fx.experimental.sym_dispatch_mode": [
+    "sym_function_mode",
+    "set_sym_function_mode"
+  ],
+  "torch.fx.experimental.sym_node": [
+    "SymNode",
```

Comment:
I take it these are all just shuffling around, not actually new?

Reply:
Yep, just good ol' shuffling + adding the new aux functions I created like `sympy_eq`.

Reply:
Call the function `_sympy_eq` so they aren't public?